### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by beenum22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ pyb install -x run_unit_tests
...
 Congratulations! You have installed Hydra. <\br >
 There is a CLI mode to get some interaction with the running tests.
+
+To make use of it. Make sure to add these two paths to your .bashrc.
+
+```
+echo 'export PATH="$HOME/hydra/src/main/scripts:$PATH"' >> ~/.bashrc
+echo 'export PYTHONPATH="$HOME/hydra/src/main/python:$PATH" >> ~/.bashrc
+source ~/.bashrc
+```
```
Comment (kratos7) [inline README.md:84]: Hi,
Can we please have some comments explaining things. Usually we expect hydra to be run from a venv
http://docs.python-guide.org/en/latest/dev/virtualenvs/

Why do you explicitly need to add python path here ?

Comment (beenum22) [inline README.md:84]: Hi,
I followed all the steps given in the guideline and in the end "hydra cli" wasn't working. I needed to add paths for it in order to make it work. And I was inside a venv as suggested in the  guideline.

**Comment by tahir24434** [comment]
@beenum22 Does this change still valid? If not, please abandon the change.

**Comment by beenum22** [comment]
@tahir24434 Yup it's not needed now. I will abandon the changes.

**Pull request closed without merging**